src/main/kotlin/com/terraformation/backend/admin/AdminDefaultVotersController.kt

```diff
@@ -64,11 +83,18 @@ class AdminDefaultVotersController(
   @PostMapping("/defaultVoters/remove")
   fun removeDefaultVoter(
       @RequestParam userId: UserId,
+      @RequestParam projectIds: List<ProjectId>?,
       redirectAttributes: RedirectAttributes,
   ): String {
+    requirePermissions { updateDefaultVoters() }
+
+    val projects = getAcceleratorProjects()
+
     try {
       defaultVoterStore.delete(userId)
       redirectAttributes.successMessage = "Default voter removed."
+
+      projectIds?.forEach { voteStore.upsert(it, getProjectPhase(it)!!, userId) }
```

Comment:
If I'm following the code correctly, when you remove a default voter and leave the checkboxes checked for their existing projects, it will keep the user on the voter list of those projects but will set their votes and conditional text to null. Do I have that right? Should this be calling `voteStore.delete()` instead?

Reply:
That's right..